admin/class-admin-init.php

```diff
@@ -401,6 +402,53 @@ private function get_yoast_seo_compatibility_notification( $name, $plugin, $leve
 		);
 	}
 
+	/**
+	 * Build Gutenberg compatibility problem notification.
+	 *
+	 * @param string $installed_version The installed version of Gutenberg.
+	 * @param string $level  			The severity level to use for the notification.
+	 *
+	 * @return Yoast_Notification The incompatible Gutenberg version notification.
+	 */
+	private function get_gutenberg_compatibility_notification( $installed_version, $level = Yoast_Notification::WARNING ) {
+		$message = sprintf(
+			/* translators: %1$s expands to Yoast SEO, %2$s expands to the installed version, %3$s expands to Gutenberg */
+			__( '%1$s detected you are using version %2$s of %3$s, please update to the latest version to prevent compatibility issues.', 'wordpress-seo' ),
+			'Yoast SEO',
+			$installed_version,
+			'Gutenberg'
+		);
+
+		return new Yoast_Notification(
+			$message,
+			array(
+				'id'   => 'wpseo-outdated-gutenberg-plugin',
+				'type' => $level,
+			)
+		);
+	}
+
+	/**
+	 * Add an alert if incompatible versions of Gutenberg are running.
+	 *
+	 * @return void
+	 */
+	public function yoast_gutenberg_compatibility_notification() {
```

Comment:
Have you considered to put all the comparing and notification logic in one class?  Like the way that has been done on: https://github.com/Yoast/wordpress-seo/blob/ef1edfd599a54e89ed442e7e8cece60968c240c2/admin/class-admin.php#L101

This keeps this extensive class cleaner and makes it easier to disable the notification without deprecate any public functionality.